src/Contrib/Jaeger/CustomizedTHttpClient.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Jaeger;
+
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+use Thrift\Exception\TTransportException;
+use Thrift\Factory\TStringFuncFactory;
+use Thrift\Transport\THttpClient;
+
+class CustomizedTHttpClient extends THttpClient
```

Comment:
I think this could just be named `HttpClient`, and you could add some docblock comments about why psr-18 was added to THttpClient...

Reply:
Yeah for sure, I went ahead and named it similarly to ThriftUdpTransport (called it THriftHttpTransport) since both files are doing the same sort of thing now (implementing TTransport).

I also renamed ThriftHttpSender to just HttpSender so it doesn't seem like it's related to those 2 files directly since that was throwing me off a little